lib/src/utils/crypto/native.dart

```diff
@@ -144,7 +144,7 @@ abstract class Cipher {
       version: version,
       initialVector: base64.encode(initialVector).replaceAll('=', ''),
       hashes: {
-        messageDigestAlgorithm: hashValue,  
+        messageDigestAlgorithm: "123",  
```

Comment:
what is the magic here?

Reply:
while testing, i forget to change it back :)). sr